### PR TITLE
added faker to seed 10 pets

### PR DESCRIPTION
**Pull request by Bradzerie**
Added faker to the seed file so we can create random pets

### CODE REVIEW

File: app/controllers/pets_controller.rb [hunk 1]
```diff
@@ -1,5 +1,20 @@
...
   end
+
+  def create
+    @pet = Pet.new(pet_params)
+    @pet.save
+  end
+
+  private
+
+  # def pet_params
```
Comment (rkeshwan) [inline app/controllers/pets_controller.rb]: please delete this for now

**Review by rkeshwan — COMMENTED** [review]
Great work!! I just left a comment for you to review. Thanks!

File: app/controllers/pets_controller.rb (hunk 1 above)
Comment (Bradzerie) [reply app/controllers/pets_controller.rb]: @rkeshwan I have deleted the code. Please review

**Review by rkeshwan — APPROVED** [review]
Nice work!!!

**Pull request merged**